_data/api-swagger.yaml

```diff
@@ -560,6 +585,112 @@ definitions:
         }
       }
       ```
+  IPAddress:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        IPAddress:
+          type: string
+          readOnly: true
+    example: |
+      An IP Address verification response may have the following content:
+      ```json
+      {
+        "Type": "IPAddress",
+        "IPAddress": "198.51.100.42"
+      }
+      ```
+  Kennisnet:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+
+  PhoneNumber:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        Number:
+          type: string
+          description: |
+            The mobile phone number of the signer.
+          example: "+31123456789"
+          required: true
+    example: |
+      A phone number verification response may have the following content:
+      ```json
+      {
+        "Type": "PhoneNumber",
+        "Number": "+31123456789"
+      }
+      ```
+
+  Scribble:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        RequireHandsignature:
+          type: boolean
+          default: false
+          description: |
+            When set the signer is required to draw a hand signature,
+            either by mouse, touchscreen or tables.
```

Comment:
When set, the signer is required to draw a scribble, either via a computer mouse, trackpad, or touchscreen.